### PR TITLE
Lakka 2.1.1

### PR DESCRIPTION
**Pull request by kivutar**

### CODE REVIEW

**Comment by Aaahh** [comment]
Give me 5 minutes

File: content/articles/2018/06/25/lakka-211-with-raspberry-pi-3-b-plus-support.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ - RetroArch 1.7.3
+ - RPi3B+
+social:
+  thumbnail: media/3bplus.jpg
+---
+
+Lakka 2.1.1 is out!
+
+This new version of Lakka retro gaming operating system now supports the new Raspberry Pi 3 B+.
+
```
Comment (ToKe79) [inline content/articles/2018/06/25/lakka-211-with-raspberry-pi-3-b-plus-support.md]: Also there was no official release for S912 yet, so worth mentioning.

File: content/articles/2018/06/25/lakka-211-with-raspberry-pi-3-b-plus-support.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - Citra, the 3DS emulator
+    - ChaiLove, the chaiscript game engine
+    - MAME2003, the multi arcade emulator
+    - PPSSPP, the PSP emulator
+    - Sameboy, the Game Boy emulator
+    - Desmume, the Nintendo DS emulator
+    - and many more 
+  - Fixes
+    - Bluetooth fix for S905/S912
+    - H3 boot
```
Comment (ToKe79) [inline content/articles/2018/06/25/lakka-211-with-raspberry-pi-3-b-plus-support.md]: Keyboard was fixed on ARM devices too (e.g. wifi password entry with physical keyboard) - was broken in 2.1 afaik.

File: nanoc.yaml [hunk 3]
```diff
@@ -69,45 +69,45 @@ data_sources:
...
+  mk808bp: http://le.builds.lakka.tv/S805.MK808B-Plus.arm/Lakka-S805.MK808B-Plus.arm-2.1.1.img.gz
+  mxq: http://le.builds.lakka.tv/S805.MXQ.arm/Lakka-S805.MXQ.arm-2.1.1.img.gz
+  S8X2.M8: http://le.builds.lakka.tv/S8X2.M8.arm/Lakka-S8X2.M8.arm-2.1.1.img.gz
+  S8X2.MXIII-1G: http://le.builds.lakka.tv/S8X2.MXIII-1G.arm/Lakka-S8X2.MXIII-1G.arm-2.1.1.img.gz
+  S8X2.MXIII-PLUS: http://le.builds.lakka.tv/S8X2.MXIII-PLUS.arm/Lakka-S8X2.MXIII-PLUS.arm-2.1.1.img.gz
+  S8X2.S82: http://le.builds.lakka.tv/S8X2.S82.arm/Lakka-S8X2.S82.arm-2.1.1.img.gz
+  S8X2.T8: http://le.builds.lakka.tv/S8X2.T8.arm/Lakka-S8X2.T8.arm-2.1.1.img.gz
+  S8X2.X8H-PLUS: http://le.builds.lakka.tv/S8X2.X8H-PLUS.arm/Lakka-S8X2.X8H-PLUS.arm-2.1.1.img.gz
+  s905: http://le.builds.lakka.tv/S905.arm/Lakka-S905.arm-2.1.1.img.gz
+  tkb: http://le.builds.lakka.tv/Rockchip.TinkerBoard.arm/Lakka-Rockchip.TinkerBoard.arm-2.1.1.img.gz
```
Comment (ToKe79) [inline nanoc.yaml]: S912 missing here, but I will rebase #51 after this is merged.

**Pull request merged**